lib/Compilation.js

```diff
@@ -1064,10 +1075,12 @@ class Compilation extends Tapable {
 			context,
 			entry,
 			module => {
+				this.hooks.buildEntry.call(module);
```

Comment:
This argument will be removed in webpack 5.
remove it.

Reply:
If I'll delete `module` then how can I get module identifier of the entry-module [here](https://github.com/webpack/webpack/pull/8279/files#diff-5d04086b4365820a456acec22b899956R169)?

Reply:
I think I got it...